app/services/container_dashboard_service.rb

```diff
@@ -100,9 +100,7 @@ def providers
 
   def build_provider_status(ui_type)
     {
-      :iconClass    => "pficon pficon-#{ui_type}",
```

Comment:
iconClass should be dealt with on the frontend side: https://github.com/zeari/manageiq/commit/e58c8bd5ac8cab149554e03143b3714adb2512a2#diff-3ab4da4c6521d98868b65b6fe274139dR91